apps/www/registry/default/ui/slider.tsx

```diff
@@ -8,17 +8,29 @@ import { cn } from "@/lib/utils"
 const Slider = React.forwardRef<
   React.ElementRef<typeof SliderPrimitive.Root>,
   React.ComponentPropsWithoutRef<typeof SliderPrimitive.Root>
->(({ className, ...props }, ref) => (
+>(({ className, orientation = "horizontal", ...props }, ref) => (
   <SliderPrimitive.Root
     ref={ref}
+    orientation={orientation}
     className={cn(
-      "relative flex w-full touch-none select-none items-center",
+      "relative flex touch-none select-none items-center",
+      orientation === "vertical" ? "flex-col h-full w-[20px]" : "w-full",
```

Comment:
we can directly use `data-[orientation="vertical"]:` here to be aligned with Radix primitives, instead of exposing `orientation` prop out

Reply:
yes we can do this way as well but I followed the approach used in the Seperator component to maintain uniformity.